### DISCUSSION TITLE
No acces dev/mem.

### DISCUSSION THREAD

**Discussion by KzalanK** [gpiozero/gpiozero#1164] [Q&A]
Hi, 
I have just strated to use my Raspberry Pi5.
My first program:
from gpiozero import LED
from time import sleep
led=LED(14)
while True:
    led.on
    sleep(1)
    led.off
    sleep(1)
But it dosn't work.
Thanks for your support

**Comment by bennuttall** [comment]
Need more information here, you've deleted the template which would have helped.

What operating system? Which pin factory? How are you running this? How did you install gpiozero?

**Comment by lurch** [comment]
GpioZero should "just work" on a Pi5, but if you've done something slightly non-standard, you need to ensure that you're using the `lgpio` pin factory https://gpiozero.readthedocs.io/en/latest/api_pins.html#changing-the-pin-factory

**Comment by RobinTail** [comment]
@KzalanK , we had two cases of that error in https://github.com/borisbu/OctoRelay/issues/347
all of them were fixed by replacing SD card.
